install/k8s/contiv/contiv.yaml

```diff
@@ -0,0 +1,257 @@
+---
+# This ConfigMap is used to configure a self-hosted Contiv installation.
```

Comment:
Do we have provision to use consul instead of etcd for contiv. I know K8s require etcd to be running. But we might need to test contiv using consul as datastore on K8s platform. I will let @jojimt  to comment on this one.